server/lib/src/server/access/mod.rs

```diff
@@ -2202,6 +2303,7 @@ mod tests {
             "name class",
             // And the class allowed is NOT an account ...
             "group",
+            "group",
```

Comment:
Ew, strings

Reply:
Yes, because those handlers take the strings and str split on them. Part of the ongoing cleanup I guess.